_layouts/base.html

```diff
@@ -8,20 +8,20 @@
     <link rel="shortcut icon" href="https://vega.github.io/favicon.ico"/>
     <meta name="description" content="{{ site.description }}">
 
-    <script src="{{site.baseurl}}/bower_components/d3/d3.min.js"></script>
-    <script src="{{site.baseurl}}/bower_components/vega/vega.min.js"></script>
+    <script src="https://d3js.org/d3.v3.min.js"></script>
```

Comment:
I just want to get rid of bower. The right way would be to compile a bundle imho but this seemed already an improvement.